lib/Cake/Test/Case/Model/Datasource/Database/MysqlTest.php

```diff
@@ -3755,6 +3755,13 @@ public function testVirtualFieldsComplexRead() {
  * @return void
  */
 	public function testIntrospectType() {
+		// There is a bug that DboSource::value() does not quote an integer or flat
```

Comment:
Isn't it related to this https://php.watch/versions/8.1/deprecate-implicit-conversion-incompatible-float-string ?

Maybe we can just remove the test if it works under PHP8?

Reply:
The new assertion failures in PHP 8.0 are in the following areas.

https://github.com/kamilwylegala/cakephp2-php8/blob/9e0fd38f37adc88e9d44debc98f485490658291c/lib/Cake/Test/Case/Model/Datasource/Database/MysqlTest.php#L3872-L3873

```
Failed asserting that two strings are equal.
Expected :'' 1.234 ''
Actual   :' 1.234 '

 /var/www/html/lib/Cake/Test/Case/Model/Datasource/Database/MysqlTest.php:3880
```

I added `markTestIncomplete()` to see if some fix is needed if the behavior has changed...
I'm a little less sure now...